README.md

```diff
@@ -19,15 +19,11 @@ If adding multiple mods, enter them in an array separated by `|`, such as `DOCKE
 
 If running CrowdSec in a container it must be on a common docker network with SWAG.
 
-Generate an API key for the bouncer with `cscli bouncers add bouncer-swag` or `docker exec -t crowdsec cscli bouncers add bouncer-swag`, if you're running CrowdSec in a container.
-
-Make a note of the API key as you can't retrieve it later without removing and re-adding the bouncer.
-
```

Comment:
I'd suggest keeping this (as it's still valid) and adding, "or can be an arbitrary string passed as the `BOUNCER_KEY_swag` environment variable in your crowdsec image", rather than adding it to the env description, which people are more likely to skip over.